Kickstarter-iOS/Views/Cells/DiscoveryPostcardCell.swift

```diff
@@ -320,6 +320,15 @@ internal final class DiscoveryPostcardCell: UITableViewCell, ValueCell {
   }
 
   @objc fileprivate func saveButtonTapped() {
+    if self.saveButton.isSelected {
+      if #available(iOS 10.0, *) {
+        self.saveButton.generateSelectionFeedback()
+      }
+    } else {
+      if #available(iOS 10.0, *) {
```

Comment:
We could consider moving the `if #available(iOS 10.0, *) { ... }` checks to inside the extension, that way we're only checking in one place for whether haptic feedback is available and calling this on pre iOS 10 will have no effect.